staging/src/k8s.io/apiserver/pkg/util/flowcontrol/default-objects.go

```diff
@@ -0,0 +1,177 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flowcontrol
+
+import (
+	rmv1a1 "k8s.io/api/flowcontrol/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// DefaultPriorityLevelConfigurationObjects returns the instances of
+// the ones defined in the KEP, with system-top in position 0 and
+// workload-low in position 1
+func DefaultPriorityLevelConfigurationObjects() []*rmv1a1.PriorityLevelConfiguration {
```

Comment:
the default objects should be moved out of generic apiserver `k8s.io/apiserver`, consider `k8s.io/kubernetes/pkg/apis/flowcontrol/v1alpha1` or a new package `k8s.io/kubernetes/pkg/registry/flowcontrol/bootstrap`

Reply:
I agree.

Reply:
I think nothing in `k8s.io/apiserver` imports from `k8s.io/kubernetes` and suspect it should stay that way.
I think perhaps the most logical allowable home is somewhere under `k8s.io/api`.